Makefile

```diff
@@ -114,10 +114,17 @@ test: examples
 		--target=thumbv7em-none-eabi --workspace
 	LIBTOCK_PLATFORM=hifive1 cargo clippy $(EXCLUDE_STD) \
 		--target=riscv32imac-unknown-none-elf --workspace
+	# example bash logic
+	if [ "${NO_RUSTUP}" = "1" ]; then
+		use_nightly_toolchain
```

Comment:
This is going to break for non-Nix users who still want to avoid using `rustup` (e.g. their system toolchains), isn't it? Also, some comments wouldn't hurt here.

Reply:
This actually doesn't do anything, because as @Samir-Rashid mentioned in his comment it doesn't change `PATH` for subsequent commands.